src/Keboola/StorageApi/Client.php

```diff
@@ -2226,10 +2231,34 @@ protected function request($method, $url, $options = array(), $responseFileName
         }
 
         if ($response->hasHeader('Content-Type') && $response->getHeader('Content-Type')[0] == 'application/json') {
-            return json_decode((string)$response->getBody(), true);
+            return json_decode((string)$this->getBody($response), true);
+        }
+
+        return (string)$this->getBody($response);
+    }
+
+    /**
+     * @param ResponseInterface $response
+     * @return string
+     * @throws ClientException
+     */
+    private function getBody(ResponseInterface $response)
+    {
+        var_export($response->getHeaderLine('Content-Encoding'));
+        $rawBody = $response->getBody()->getContents();
+
+        // Curl does not support Brotli compression - use Brotli extension
+        if (extension_loaded('brotli') && $response->getHeaderLine('Content-Encoding') === 'br') {
```

Comment:
lepší by bylo tady naimplementovat in-place dekodování v té response a třeba setnou zpátky to body... nebo tak něco.

Reply:
vyreseno pomoci `BrotliStream`